interoperability_report.py

```diff
@@ -841,12 +901,16 @@ def main():
     timeout = 10
     now = datetime.now()
     for k, v in rtps_test_suite_1.items():
```

Comment:
this way we can only use one dictionary at a time. We should find a way of using different dictionaries (so far is ok but we should look at this in the future)